docs/scripts/readme_plot.py

```diff
@@ -67,45 +79,48 @@ class MoneyModelDF(ModelDF):
     def __init__(self, N):
         super().__init__()
         self.num_agents = N
-        self.create_agents(N, {MoneyAgentDF: 1})
+        self.agents = self.agents.add(MoneyAgentsDF(N, model=self))
```

Comment:
Shouldn't this be an inplace operation, based on the API of `AgentSet`?
```python
self.agents.add(...)
# instead of
self.agents = self.agents.add(...)
```

Reply:
Is it because Polars doesn't support inplace operations?